addons/stock/models/stock_rule.py

```diff
@@ -666,7 +665,7 @@ def _get_push_rule(self, product_id, location_dest_id, values):
             domain = [('location_src_id', '=', location.id), ('action', 'in', ('push', 'pull_push'))]
             if values.get('domain'):
                 domain = expression.AND([domain, values['domain']])
-            found_rule = self._search_rule(values.get('route_ids'), values.get('product_packaging_id'), product_id, values.get('warehouse_id'), domain)
+            found_rule = self._search_rule(values.get('route_ids'), values.get('uom_id'), product_id, values.get('warehouse_id'), domain)
```

Comment:
To keep things consistent with what's its supposed to represent, might be better to call il `package_uom_id` instead of `uom_id` (same applies to other places where rules are searched). Otherwise it will raise confusion on *which* uom is used